### PR TITLE
A few improvements suggested by @beaufortfrancois

### PR DESCRIPTION
**Pull request by jankeromnes**
Thanks François! 😄 💯

### CODE REVIEW

**Comment by jankeromnes** [comment]
@nt1m Please have a look if you have some time, I'd like your opinion on these small changes.

Also, it's much easier to review `templates/contributions.html` [with ?w=1 in the URL](https://github.com/JanitorTechnology/janitor/pull/164/files?w=1) in order to ignore noisy whitespace changes.

Thanks!

File: templates/contributions.html [hunk 1]
```diff
@@ -43,16 +43,13 @@ <h2 class="header">My Contributions</h2>{{
               <a class="btn btn-primary"{{
                 if (updated < c9sdkStart) { }} href="https://c9.io/open/ssh?name={{= projectId in id}}-{{= id in id}}&description={{= project.name in uri}}%20on%20janitor.technology&host={{= machine.docker.host in uri}}&port={{= machine.docker.ports['22'].port in integer}}&user=user&workspaceDir={{= project.docker.path in uri}}&nodePath=%2Fhome%2Fuser%2F.c9%2Fnode%2Fbin%2Fnode" {{ }
                 else { }} href="https://{{= machine.docker.host in uri}}/{{= machine.docker.container.slice(0,16) in uri}}/8089/ide.html" {{ }
-              }}target="_blank"><span class="hidden-xs">Edit in </span>Cloud9</a>
+              }}target="_blank"><span class="hidden-xs">Open </span>IDE</a>
```
Comment (nt1m) [inline templates/contributions.html]: I like this change :+1:

**Comment by beaufortfrancois** [comment]
👍

**Comment by jankeromnes** [comment]
Alright, thanks everyone! Let's merge and deploy this. 😄

**Comment by beaufortfrancois** [comment]
LGTM. Thank you!

**Pull request merged**